coding/javascript.md

```diff
@@ -70,6 +70,14 @@ To ensure code formatting consistency across our codebase, we use [Prettier](htt
 
 JavaScript should also be written using [AirBnB](https://github.com/airbnb/javascript) style. New projects can adhere to this by selecting 'AirBnB' during `eslint init`, and existing projects can find the relevant ESLint config in [eslint-config-airbnb](https://www.npmjs.com/package/eslint-config-airbnb).
 
+### Commenting code
+
+As a general rule, human-readable code is preferred over brevity.
```

Comment:
This is a general coding principal, but I'm not sure it belongs under "Commenting code".

Reply:
I think that this can be rewritten to get across the idea that if code is so "clever" that it needs comments to document it then the code should be rewritten in an easier to read style.

Reply:
Perhaps this can be hoisted into the section above `Code style & formatting`?